content/docs/state-and-lifecycle.md

```diff
@@ -246,21 +246,22 @@ The `componentDidMount()` method runs after the component output has been render
   }
 ```
 
-Note how we save the timer ID right on `this`.
 
-While `this.props` is set up by React itself and `this.state` has a special meaning, you are free to add additional fields to the class manually if you need to store something that doesn’t participate in the data flow (like a timer ID).
+ध्यान दें कि हम टाइमर आईडी को `this` पर सही तरीके से सहेजते हैं।
```

Comment:
`आईडी` => `ID`

Reply:
`(एक टाइमर आईडी की तरह)।` => `(एक टाइमर ID की तरह)।`

can you update the ID for this also?